site/docs/fargate-tutorials/django.md

```diff
@@ -180,59 +180,36 @@ If the function call is successful, it will return a list of dicts with each clo
 
 ## Login, build and push your code to the container registry
 
-1. Grab your new `ECR URI` from the hosted DB
+Previously, you needed to manually build and push your image to the ECR. But recently we've added the high-level `ecr_build` SQL function which does all those steps automatically. It will do the following:
+- Pull the codes from your Github repository
```

Comment:
Nit. `code` is one of those weird words in English that has still has a remnant rule on pluralization, like `fish`.

| Pluralization | `code` | `fish` |
| ------------- | ------ | ------ |
| Singular      | `code` | `fish` |
| Plural of same type | `code` | `fish` |
| Plural of multiple types | `codes` | `fishes` |

Making this:

```suggestion
- Pull the code from your Github repository
```

Because it's all code from the same repo.